packages/wallets/src/components/DerivAppsSection/DerivAppsTradingAccount.tsx

```diff
@@ -37,7 +37,12 @@ const DerivAppsTradingAccount: React.FC = () => {
             <WalletButton
                 color='white'
                 onClick={() => {
-                    history.push('wallets/cashier/transfer');
+                    history.push(
+                        '/wallets/cashier/transfer',
+                        activeLinkedToTradingAccount?.loginid
+                            ? { toAccountLoginId: activeLinkedToTradingAccount?.loginid }
+                            : undefined
+                    );
```

Comment:
Have added this to make this button work with target transfer account pre-selection (as shown in the 3rd description video)

<img width="1792" alt="Screenshot 2024-01-26 at 18 57 00" src="https://github.com/binary-com/deriv-app/assets/119863957/45ad8891-c1ba-42dc-9459-ea0b5a2d6da7">